pkg/ovsdb/handler.go

```diff
@@ -198,7 +226,12 @@ func (ch *Handler) Steal(ctx context.Context, param interface{}) (interface{}, e
 
 func (ch *Handler) MonitorCond(ctx context.Context, param ovsjson.CondMonitorParameters) (interface{}, error) {
 	klog.V(5).Infof("MonitorCond request, parameters %v", param)
-
+	ch.mu.Lock()
+	defer ch.mu.Unlock()
+	if ok := ch.createOrDieMonitorsArray(param.JsonValue.(string)); ok != nil {
+		return nil, fmt.Errorf("duplicate json-value")
```

Comment:
use the error return by `createOrDieMonotirsArray`

Reply:
done